DOL.WHD.Section14c.Log/Repositories/ActivityLogRepository.cs

```diff
@@ -0,0 +1,54 @@
+﻿using NLog;
+using System;
+using System.Linq;
+using System.Threading.Tasks;
+using System.Web;
+using System.Web.Http;
+using DOL.WHD.Section14c.Log.Models;
+using DOL.WHD.Section14c.Log.DataAccess;
+
+namespace DOL.WHD.Section14c.Log.DataAccess.Repositories
+{
+    /// <summary>
+    /// Activity Log Repository
+    /// </summary>
+    public class ActivityLogRepository : IActivityLogRepository
+    {
+        private readonly ApplicationLogContext _dbContext;
+        private static readonly Logger _logger = LogManager.GetCurrentClassLogger();        
+        
+        private Boolean Disposed;
+
+        /// <summary>
+        /// Activity Log Repository
+        /// </summary>
+        public ActivityLogRepository()
+        {
+            _dbContext = new ApplicationLogContext();
+        }
+
+        /// <summary>
+        /// Get ALl Logs
+        /// </summary>
+        /// <returns></returns>
+        public IQueryable<APIActivityLogs> GetAllLogs()
+        {
+            return _dbContext.ActivityLogs.AsQueryable();
+        }
+
+        /// <summary>
+        /// Dispose Object
+        /// </summary>
+        public void Dispose()
```

Comment:
There are a lot of warnings in the build log about `Dispose()` implementations throughout the code.  Not anything that needs fixed now, but I'm going to spend a little time understanding the warnings and whether they need any code changes.  If y'all already know we can ignore them, just let me know.  😄

Reply:
@mgwalker what is the warning?